packages/cockroach/buildinfo.json

```diff
@@ -1,9 +1,9 @@
 {
+  "requires": ["ncurses"],
   "single_source" : {
-      "kind": "git",
-      "git": "git@github.com:cockroachdb/cockroach.git",
-      "ref": "486632793834b3fded5cf8bc87dddf0041f2297e",
-      "ref_origin": "v1.1.9"
+      "kind": "url_extract",
```

Comment:
Why have you changed from Github to binary download?

Reply:
Its not actually a binary, but rather source code from which we build the cockroachdb.

Reply:
OK, but why the change?